### PR TITLE
[PAGOPA-831] payment integrazione con gpd core

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

- added: `paSendRTV2` and `paGetPaymentV2`
- fix integration test
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jacopocarlini** [comment]
update_code

File: api-test/DEV.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"id": "4582c6a3-ae67-41b8-b577-a2c6acd27a38",
+	"name": "DEV",
+	"values": [
+		{
+			"key": "host",
+			"value": "https://api.dev.platform.pagopa.it/afm/calculator-service/v1",
```
Comment (andrea-deri) [inline api-test/DEV.postman_environment.json:7]: is this value a typo or must it reference to AFM?

File: integration-test/src/step_definitions/support/clients/api_config_client.js [hunk 2]
```diff
@@ -6,47 +6,47 @@ const api_config_host = process.env.api_config_host;
 function apiConfigHealthCheck() {    
     return get(api_config_host + `/info`, {
         headers: {
-            "Ocp-Apim-Subscription-Key": process.env.APICONFIG_SUBSCRIPTION_KEY
+            "Ocp-Apim-Subscription-Key": process.env.SUBKEY
```
Comment (andrea-deri) [inline integration-test/src/step_definitions/support/clients/api_config_client.js:9]: Correct me if i'm wrong: those SUBKEY env parameters are passed as GH env variable, are they?

**Review by andrea-deri — APPROVED** [review]
Good Job! 👍🏻

File: integration-test/src/step_definitions/support/clients/api_config_client.js (hunk 2 above)
Comment (jacopocarlini) [reply integration-test/src/step_definitions/support/clients/api_config_client.js:9]: yes

File: api-test/DEV.postman_environment.json (hunk 1 above)
Comment (jacopocarlini) [reply api-test/DEV.postman_environment.json:7]: it is an error

**Review by andrea-deri — CHANGES_REQUESTED** [review]

File: .github/workflows/integration_test.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ jobs:
   integration_test:
     name: Test
     runs-on: ubuntu-latest
-    environment: ${{( github.event.inputs == null && 'dev') || inputs.environment }}
+    environment: ${{( (github.event.inputs == null || inputs.environment == 'local') && 'dev') || inputs.environment }}
```
[the remark this answers is not in the document]
Comment (jacopocarlini) [reply .github/workflows/integration_test.yml]: done

**Comment by jacopocarlini** [comment]
update_code

**Comment by jacopocarlini** [comment]
update_code

**Review by andrea-deri — APPROVED** [review]
Great job!

**Pull request merged**